torchci/lib/bot/autoLabelBot.ts

```diff
@@ -113,9 +115,11 @@ const notUserFacingPatternExceptions: RegExp[] = [
 // ]
 const repoSpecificAutoLabels: {[repo: string]: [RegExp, string][]}  = {
   // Sample entry below
-  // "pytorch/pytorch": [
-  //    [/torch\/(csrc|cpp)\/api/gi, "release notes: cpp"]
-  // ],
+  "pytorch/pytorch": [
+      [/aten\/src\/ATen\/mps/gi, , "ciflow/mps"],
+      [/aten\/src\/ATen\/native\/mps/gi, , "ciflow/mps"],
+      [/test\/test_mps.py/gi, , "ciflow/mps"],
```

Comment:
```suggestion
      [/aten\/src\/ATen\/mps/gi, "ciflow/mps"],
      [/aten\/src\/ATen\/native\/mps/gi, "ciflow/mps"],
      [/test\/test_mps.py/gi, "ciflow/mps"],
```
copy paste mistake?

also could you add a test case :)

Reply:
I'm not sure we want to test repo specific rules here